ui/pages/confirmations/components/confirm/info/shared/transaction-details/transaction-details.tsx

```diff
@@ -83,6 +86,30 @@ export const MethodDataRow = () => {
   );
 };
 
+export const AmountRow = () => {
```

Comment:
Do we need to export this if it's only used locally?

Reply:
Done https://github.com/MetaMask/metamask-extension/pull/27594/commits/b2ff77a324f3e386cf78b8b99f8bae25435c91f5